skore/src/skore/sklearn/_cross_validation/report.py

```diff
@@ -0,0 +1,302 @@
+import time
+
+import joblib
+import numpy as np
+from sklearn.base import clone, is_classifier
+from sklearn.model_selection import check_cv
+from sklearn.pipeline import Pipeline
+
+from skore.externals._pandas_accessors import DirNamesMixin
+from skore.externals._sklearn_compat import _safe_indexing
+from skore.sklearn._base import _BaseReport
+from skore.sklearn._estimator.report import EstimatorReport
+from skore.sklearn.find_ml_task import _find_ml_task
+from skore.utils._progress_bar import progress_decorator
+
+
+def _generate_estimator_report(estimator, X, y, train_indices, test_indices):
+    return EstimatorReport(
+        estimator,
+        fit=True,
+        X_train=_safe_indexing(X, train_indices),
+        y_train=_safe_indexing(y, train_indices),
+        X_test=_safe_indexing(X, test_indices),
+        y_test=_safe_indexing(y, test_indices),
+    )
+
+
+class CrossValidationReport(_BaseReport, DirNamesMixin):
+    """Reporter for cross-validation results.
+
+    Parameters
+    ----------
+    estimator : estimator object
+        Estimator to make report from.
+
+    X : {array-like, sparse matrix} of shape (n_samples, n_features)
+        The data to fit. Can be for example a list, or an array.
+
+    y : array-like of shape (n_samples,) or (n_samples, n_outputs), default=None
+        The target variable to try to predict in the case of supervised learning.
+
+    cv_splitter : int, cross-validation generator or an iterable, default=5
+        Determines the cross-validation splitting strategy.
+        Possible inputs for `cv_splitter` are:
+
+        - int, to specify the number of folds in a `(Stratified)KFold`,
+        - a scikit-learn:term:`CV splitter`,
```

Comment:
I believe you are trying to provide a link to the scikit-learn glossary?

Reply:
The linking works (at least locally) so I have good hope. Missing a space thought.

```suggestion
        - a scikit-learn :term:`CV splitter`,
```